src/index.js

```diff
@@ -108,8 +151,11 @@ class MaterielnetKonnector extends CookieKonnector {
                             body = JSON.parse(body);
                         }
                         if (!body || !body.authenticationSuccess || !body.user) {
-                            if (typeof(body.loginForm) === "string" && body.loginForm.includes("window.renderCaptcha()")) {
+                            if (typeof(body.loginForm) === "string" &&
+                                body.loginForm.includes("window.renderCaptcha()")) {
```

Comment:
Maybe we should use a constant to make it easier to modify it later if needed?

Reply:
Ok, I remove this type testing too, as it is not necessary anymore.